### PR TITLE
user page

### PR DESCRIPTION
**Pull request by philipkcl**

### CODE REVIEW

File: core/templates/core/user_expanded_search_table_layout.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    {% for r in search_results %}
+        <tr class="selectable_entry {% if forloop.counter|divisibleby:2 %}even{% else %}odd{% endif %}"
+            entry_id="{{ r.pk }}">
+
+            <td>
+                {% include 'core/component/search_table_layout/link_record.html' with id=r.pk view='user:full_form' %}
+            </td>
+            <td>{{ r.email }}</td>
+            <td>{{ r.surname }} {{ r.forename }}</td>
```
Comment (J4bbi) [inline core/templates/core/user_expanded_search_table_layout.html:28]: Just missing a comma between surname and forename

**Review by J4bbi — CHANGES_REQUESTED** [review]
Looks good.

What is missing is an ability to reset a forgotten or lost password. This is not currently possible I believe except for logging in to Django admin interface and doing it there.

File: core/templates/core/user_expanded_search_table_layout.html (hunk 1 above)
Comment (philipkcl) [reply core/templates/core/user_expanded_search_table_layout.html:28]: where the comma come from, original website seem have no comma for surname and forename

**Comment by philipkcl** [comment]
since `reset a forgotten or lost password` to be confirm the requirement, let's merge this branch first

**Pull request merged**